tests/metrics/test_observers.py

```diff
@@ -1,9 +1,42 @@
 from collections import namedtuple
 
+import numpy as np
 import pandas as pd
+import pytest
+from vivarium import InteractiveContext
+from vivarium.testing_utilities import TestPopulation, build_table
 
-from vivarium_public_health.disease import DiseaseState, RiskAttributableDisease
-from vivarium_public_health.metrics.disability import DisabilityObserver
+from vivarium_public_health.disease import (
+    DiseaseModel,
+    DiseaseState,
+    RiskAttributableDisease,
+)
+from vivarium_public_health.disease.state import SusceptibleState
+from vivarium_public_health.metrics.disability import (
+    DisabilityObserver as DisabilityObserver_,
+)
+from vivarium_public_health.metrics.stratification import (
+    ResultsStratifier as ResultsStratifier_,
+)
+
+
+class ResultsStratifier(ResultsStratifier_):
```

Comment:
Since you're just using these in tests, why don't you make a ResultsStratifier fixture and a DisabilityObserver fixture and set the configuration defaults for each of those in the fixture. i.e.
```
@pytest.fixture
def results_stratifier():
    stratifier = ResultsStratifier()
    stratifier.configuration_defaults = {...}
```

That seems more idiomatic than subclassing these classes.

Reply:
The intent was to use the Observer and ResultsStratifier as would be used in a model repository, which is subclassing as far as I understand. I didn't mean to use the subclasses for the pre-existing unit tests however, and ought to switch those to use the vanilla version of the DisabilityObserver.

Reply:
That's fine